packages/accordions/src/elements/components/Step.spec.tsx

```diff
@@ -0,0 +1,50 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import React from 'react';
+import { render } from 'garden-test-utils';
+import { Stepper } from '../Stepper';
+import { Step } from './Step';
+
+describe('Step', () => {
+  it('passes ref to underlying DOM element', () => {
+    const ref = React.createRef<HTMLDivElement>();
+    const { getByTestId } = render(
+      <Stepper>
+        <Step data-test-id="step" ref={ref} />
+      </Stepper>
+    );
+
+    expect(getByTestId('step')).toBe(ref.current);
```

Comment:
This ref test needs to be against `Stepper.Step` in order to be valid.

Reply:
Whoops, this isn't how the API is used. Thanks for the suggestion, I've updated the tests to use the `Stepper` sub-components where necessary 👍 .